README.md

```diff
@@ -230,36 +230,36 @@ export interface PluginOptions {
   rollupTypes?: boolean
 
   /**
-   * Whether copy .d.ts source files into outputDir
+   * Whether to copy .d.ts source files to outputDir
    *
    * @default false
-   * @remarks Before 2.0 it defaults to true
+   * @remarks Prior to 2.0, the default was true
    */
   copyDtsFiles?: boolean
 
   /**
-   * Whether emit nothing when has any diagnostic
+   * Whether to emit nothing when there is a diagnostic
    *
    * @default false
    */
   noEmitOnError?: boolean
 
   /**
-   * Whether skip typescript diagnostics
+   * Whether to skip Typescript diagnostics
    *
-   * Skip type diagnostics means that type errors will not interrupt the build process
+   * Skip type diagnostics means type errors will not interrupt the build process
    *
-   * But for the source files with type errors will not be emitted
+   * But source files with type errors will not be emitted
    *
    * @default false
-   * @remarks Before 1.7 it defaults to true
+   * @remarks Before 1.7, default was true
    */
   skipDiagnostics?: boolean
 
   /**
-   * Customize typescript lib folder path
+   * Customize Typescript lib folder path
    *
-   * Should pass a relative path to root or a absolute path
+   * SRelative path to root or an absolute path
```

Comment:
Did the `S` miss to be removed?

Reply:
Looks like a typo on my part! The first "S" should be removed.

Reply:
Nice catch - I just pushed an update to fix that.